src/service/standard-actions-impl.js

```diff
@@ -267,6 +269,33 @@ export class StandardActions {
       return this.handleHide(invocation);
     }
   }
+
+  /**
+   * Handles "toggleClass" action.
+   * @param {!./action-impl.ActionInvocation} invocation
+   * @return {?Promise}
+   */
+  handleToggleClass(invocation) {
+    if (!invocation.satisfiesTrust(ActionTrust.HIGH)) {
+      return null;
+    }
+
+    const target = dev().assertElement(invocation.node);
+    const {args} = invocation;
+    const className = user().assertString(args['class']);
```

Comment:
let's provide a nice error message if `class` was omitted. second arg to `assertString` can be an error message. Something like `No class argument provided for toggleClass action`